[charts] Export charts as image (#17353)
mui/mui-x: docs/data/charts/export/export.md

{{"demo": "PrintChart.js"}}

## Export as image

> The `apiRef` prop also exposes a `exportAsImage` function that can be used to export the chart as an image.

Reviewer: ```suggestion
The `apiRef` prop also exposes a `exportAsImage()` method to export the chart as an image.
```

cf: **Referencing JS identifiers** section in https://mui-org.notion.site/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b

> For it to work, you need to add `rasterizehtml` as a dependency in your project's `package.json`.

Reviewer: Just a proposal

`````suggestion
### Dependency

For it to work, you need to add `rasterizehtml` as a dependency in your project.

<codeblock storageKey="package-manager">

```bash npm
npm install rasterizehtml
```

```bash pnpm
pnpm add rasterizehtml
```

```bash yarn
yarn add rasterizehtml
```

</codeblock>

### Usage
`````

The function accepts an options object with the `type` property, which specifies the image format. The available formats are:

- `image/png` and `image/jpeg`, which are supported across all [supported platforms](/material-ui/getting-started/supported-platforms/);
- `image/webp` which is only supported in some browsers.

> If you provide a format that your browser does not support, `exportAsImage` will fall back to `image/png`.

Reviewer: ```suggestion
If the format is not supported by the browser, `exportAsImage()` falls back to `image/png`.
```
